src/pages/namju/Login/Login.js

```diff
@@ -1,8 +1,87 @@
-import React from 'react';
-import './Login';
+import React, { useState, useEffect } from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+import './Login.scss';
 
-const Login = () => {
-  return <div></div>;
-};
+function Login() {
+  const [inputValid, setInputValid] = useState(false);
+  const [inputValues, setInputValues] = useState({
+    userName: '',
+    password: '',
+  });
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setInputValues({
+      ...inputValues,
+      [name]: value,
+    });
+  };
+
+  useEffect(() => {
+    setInputValid(
+      inputValues.userName.includes('@') && inputValues.password.length >= 5
+    );
+  }, [inputValues]);
+
+  const navigate = useNavigate();
+  const goToMain = () => {
+    fetch('http://10.58.2.159/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: inputValues.userName,
+        password: inputValues.password,
+        // account: inputValues.userName,
+        // name: inputValues.userName,
+        // phone: '010-2937-1623',
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.message === 'USER_DOES_NOT_EXIST') {
+          alert('잘못입력했으니 확인 부탁');
+        } else if (result.message === 'INVALID_PASSWORD') {
+          alert('비번 확인해라!!!');
+        } else {
+          alert('냠냠굿~');
+          navigate('/main-namju');
```

Comment:
에러를 객체 형태로 맵핑해서 관리하면 유지보수, 가독성 측면에서 좋습니다.
```
const ERRORS = {
   USERS_DOES_NOT_EXIST: '잘못입력했으니 확인 부탁',
   INVALID_PASSWORD: '비번 확인해라!!!',
}
```
```suggestion
        if (result.access_token) {
          alert('냠냠굿~');
          navigate('/main-namju');
          return;
        }
        alert(ERRORS[result.message]);
```

Reply:
감사합니다 냠냠굿~